### ISSUE TITLE
1.0.0 release

### ISSUE THREAD

**Issue by sgillies** [mapbox/supermercado#30]
This package is super useful and deserves a 1.0.0 release. It's quite close! Maybe early in 2020 @dnomadb?

**Comment by dnomadb** [comment]
Yes @sgillies ! There are quite a few tickets / ideas in my mind that should be improved:
- Support for all geometry types #29 #28 
- Formatting, docstrings, etc
- Merge #26
